src/ipc/ipc3/handler.c

```diff
@@ -430,10 +430,10 @@ static int ipc_stream_trigger(uint32_t header)
 		cmd = COMP_TRIGGER_STOP;
 		break;
 	case SOF_IPC_STREAM_TRIG_PAUSE:
-		cmd = COMP_TRIGGER_PAUSE;
+		cmd = COMP_TRIGGER_STOP;
 		break;
 	case SOF_IPC_STREAM_TRIG_RELEASE:
-		cmd = COMP_TRIGGER_PRE_RELEASE;
+		cmd = COMP_TRIGGER_PRE_START;
```

Comment:
any issues with this ?

Reply:
We want to remove PAUSE and RELEASE, so both RELEASE and PRE_RELEASE should go